README.md

```diff
@@ -1,2 +1,3 @@
 Test
 ====
+I have no idea how to code or program, but I'm going to learn so that I'll be able to actualy understand what's going on
```

Comment:
I have no idea how to code or program, but I'm going to learn so that I'll be able to do stuff